### PR TITLE
RDM-13015 FTs for the new CaseAccessCategory field in the Global Search endpoint

### PR DESCRIPTION
**Pull request by marianadpereira**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDM-13015

### Change description ###

FTs for the new CaseAccessCategory field in the Global Search endpoint.

This is also enabling F-1005 and F-1006.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -19,7 +19,7 @@ def branchesToSync = ['demo', 'ithc', 'perftest']
 
 // Variables to switch pipeline logic and wiring per type of build
 
-def definitionStoreDevelopPr = "PR-575"  // This doesn't change frequently, but when it does, only change this value.
+def definitionStoreDevelopPr = "PR-1170"  // This doesn't change frequently, but when it does, only change this value.
```
Comment (marianadpereira) [inline Jenkinsfile_CNP:22]: This needs to reverted before merging

File: Jenkinsfile_CNP [hunk 2]
```diff
@@ -94,7 +94,7 @@ env.CASE_DOCUMENT_AM_URL = "http://ccd-case-document-am-api-aat.service.core-com
 env.RD_LOCATION_REF_API_BASE_URL = "http://rd-location-ref-api-aat.service.core-compute-aat.internal"
 env.RD_PROFESSIONAL_API_BASE_URL = "http://rd-professional-api-aat.service.core-compute-aat.internal"
 env.BEFTA_RESPONSE_HEADER_CHECK_POLICY="JUST_WARN"
-env.ELASTIC_SEARCH_FTA_ENABLED = "false"
+env.ELASTIC_SEARCH_FTA_ENABLED = "true"
```
Comment (marianadpereira) [inline Jenkinsfile_CNP:97]: This needs to be reverted before merging

File: build.gradle [hunk 3]
```diff
@@ -264,8 +264,8 @@ dependencies {
     compile 'org.jooq:jool-java-8:0.9.14'
     compile 'com.github.hmcts:ccd-case-document-am-client:1.7.1'
 
-    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.13.5-prerelease'
-    testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.1'
+    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.14.1-prerelease'
+    testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.2-Categories'
```
Comment (marianadpereira) [inline build.gradle:268]: This needs to be updated to a full release once def file is merged https://github.com/hmcts/ccd-test-definitions/pull/41

File: charts/ccd-data-store-api/values.preview.template.yaml [hunk 4]
```diff
@@ -23,7 +23,7 @@ java:
     DATA_STORE_DB_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
     DATA_STORE_DB_OPTIONS: "?stringtype=unspecified"
     DATA_STORE_DB_MAX_POOL_SIZE: 10
-    DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-575.service.core-compute-preview.internal/
+    DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-1170.service.core-compute-preview.internal/
```
Comment (marianadpereira) [inline charts/ccd-data-store-api/values.preview.template.yaml:26]: This needs to be reverted before merging

**Review by sullivangareth — APPROVED** [review]

**Review by mattnayler — APPROVED** [review]

File: build.gradle (hunk 3 above)
Comment (mattnayler) [reply build.gradle:268]: can this be updated now that the test-definitions PR has been merged?

Comment (marianadpereira) [reply build.gradle:268]: Updated

**Pull request merged**